ansible.cfg

```diff
@@ -1,2 +1,3 @@
 [defaults]
 roles_path = ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:roles
+lookup_plugins = ~/.ansible/plugins/lookup:/usr/share/ansible/plugins/lookup:~/pulp_installer/plugins/lookup
```

Comment:
Do we use this file? Do the molecule tests rely on this?

Reply:
I tried and molecule did not use it, I had to set a variable in molecule